lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -173,7 +172,7 @@ def get_all_dicom_treatment_info(dicomFile):
             except (TypeError, ValueError, AttributeError):
                 dicom_beam["tolerance"] = 0
 
-            if dicom_beam["manufacturer"] == "Varian":
+            if dicom_beam["machine"] == "Vault 1-IMRT":
```

Comment:
It'll be worth sorting out using the config file for the sorts of things. It might be worth running through how to set up these items in a config file with you on a video call at some point.

Reply:
Yes, I agree. I have been trying to keep this in mind, but definitely taken the easy route at some points.

Reply:
Easy route is perfectly fine to get something up and running 🙂